src/i18n.tsx

```diff
@@ -1,11 +1,9 @@
 import { SupportedLocale } from 'constants/locales'
-import { initialLocale, useActiveLocale } from 'hooks/useActiveLocale'
-import { dynamicActivate, Provider } from 'lib/i18n'
+import { useActiveLocale } from 'hooks/useActiveLocale'
+import { Provider } from 'lib/i18n'
 import { ReactNode, useCallback } from 'react'
 import { useUserLocaleManager } from 'state/user/hooks'
 
-dynamicActivate(initialLocale)
```

Comment:
a bit scary to have changes like this but i assume we had to change some sort of locale strategy due to swc? just curious why

Reply:
I agree - this is because swc forced us to updatel ingui, which removed some of the functionality that we'd used.